addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -419,6 +419,9 @@ export const editorCommands = {
         for (const node of getTraversedNodes(editor.editable)) {
             // The only possible background image on text is the gradient.
             closestElement(node).style.backgroundImage = '';
+            if (node.nodeName === "FONT" || node.nodeName === "SPAN") {
+                node.parentNode.replaceChild(document.createTextNode(node.textContent), node);
```

Comment:
This is **way** too drastic !


You cannot just replace the element with nothing.
* If one of the element has an attribute -> you cannot remove it
* If one of the element is a font-awesome icon -> you cannot remove it 
    => I've tested and encounter this issue is website !
* If one of the element has a css class or id that is not used to add background or text color -> you need to keep it